osquery/logger/plugins/tls_logger.cpp

```diff
@@ -58,6 +60,43 @@ Status TLSLoggerPlugin::logStatus(const std::vector<StatusLogLine>& log) {
   return forwarder_->logStatus(log);
 }
 
+Status TLSLoggerPlugin::logEvent(const std::string& s) {
+  std::string capture_event_types = Flag::getValue("logger_tls_event_types");
```

Comment:
There are two thread which can access this variable (FLAGS_logger_tls_event_types, string type) -
1. logEvent got called, this variable is being accessed (read access).
2. config is being refreshed and new value is being set(write access) for this variable.

Clearly std::string is not an atomic type need to be accessed under the protection of a lock. 
Thats why the APIs 
flags::GetCommandLineOption and flags::SetCommandLineOption for thread safe access.